x-pack/plugins/cloud_security_posture/server/lib/telemetry/collectors/installation_stats_collector.ts

```diff
@@ -71,7 +71,7 @@ export const getInstallationStats = async (
   });
   if (!packagePolicies) return [];
 
-  const installationStats: CloudSecurityInstallationStats[] = await getInstalledPackagePolicies(
+  const installationStats: CloudSecurityInstallationStats[] = getInstalledPackagePolicies(
```

Comment:
`getInstalledPackagePolicies` is an async method - so it returns a promise and not `CloudSecurityInstallationStats[]`
Either revert the change or 

```
return getInstalledPackagePolicies(packagePolicies.items, agentPolicies?.items || []);
```

Reply:
Hey Kfir, I added async/await to  `getInstalledPackagePolicies` which returns an array.  @JordanSh  
Pointed out async/await was unnecessary which is true since   `getInstalledPackagePolicies` return an array.  The `getInstalledPackagePolicies ` returns an array, not a promise.
<img width="1063" alt="image" src="https://github.com/elastic/kibana/assets/17135495/19f71498-1f6d-4960-9358-86774d897f07">